module/Makefile.in

```diff
@@ -90,7 +90,11 @@ modules_install-Linux:
 	if [ -n "$(DESTDIR)" ]; then \
 		find $$kmoddir -name 'modules.*' | xargs $(RM); \
 	fi
+	@# Debian ships sub-1kb fake System.map files that just say
+	@# "if you want system.map go install this package"
+	@# Naturally, depmod is less than amused by this.
 	sysmap=$(INSTALL_MOD_PATH)/boot/System.map-@LINUX_VERSION@; \
+	[ -z "$$(find $$sysmap -type f -size +1024c)" ] && sysmap=$(INSTALL_MOD_PATH)/usr/lib/debug/boot/System.map-@LINUX_VERSION@; \
```

Comment:
nit: >80 columns

Reply:
I don't see a way to fit "sysmap=$(INSTALL_MOD_PATH)/usr/lib/debug/boot/System.map-@LINUX_VERSION@;", since that's 74 characters on its own, without blowing 80c, unless I go add a number of variable expansions solely to avoid that.

It looks like in other places in e.g. /Makefile.in, if the line needs to blow 80c, it's just permitted to.

Reply:
(Sorry for the noise, I accidentally pushed an old version when I failed to copy the changes I made locally back from the testbed...should be good now.)

Reply:
Close is good enough for the Makefiles!  Your updated version works for me.